docs/source/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -324,8 +324,37 @@ Individual intervention pages:
      - None 
      - 
 
-.. todo::
-  Discuss appropriate number of seeds and draws  
+
+**Final Model Run:**
+
+.. list-table:: CVD simulation model population parameters
+   :header-rows: 1
+
+   * - Parameter
+     - Value
+     - Note
+   * - Population size
```

Comment:
This is not how it's structured in the template, but I think @aflaxman prefers to have "population size per draw" and "number of draws" defined by the researcher and "number of random seeds per draw" and "population size per random seed" defined by the engineers (so that the total population size matches what you've specified). Maybe we can edit this table so that it doesn't specify the number of random seeds to be in accordance with that preference and then eventually update the template too so we don't run into this again!

Reply:
Okay, that makes sense from our chat yesterday. Thanks!